project/cmake/addons/CMakeLists.txt

```diff
@@ -132,7 +132,9 @@ endif()
 get_filename_component(ADDONS_DEFINITION_DIR "${ADDONS_DEFINITION_DIR}" ABSOLUTE)
 
 if(ADDON_SRC_PREFIX)
-  get_filename_component(ADDON_SRC_PREFIX "${ADDON_SRC_PREFIX}" ABSOLUTE)
+  if(NOT IS_ABSOLUTE ${ADDON_SRC_PREFIX})
+    message(FATAL_ERROR "ADDON_SRC_PREFIX was an relative path, which is not supported! Please use a absolute path for ADDON_SRC_PREFIX.")
```

Comment:
@akva2 
Well nothing. I'm fine with this solution. But if you ask 3 developers you get 3 meanings where the relative paths points to. So we decided to throw an error on relative paths.